### PR TITLE
Updates to Automations and Notifications documentation

### PR DESCRIPTION
**Pull request by tpdorsey**
Updates Automations documentation to include notification actions, including templated notification messages.

Adds note to Notifications that functionality is provided by Automations in Prefect Cloud.

### Example

Preview:
- [Automations](https://deploy-preview-8140--prefect-orion.netlify.app/ui/automations/)
- [Notifications](https://deploy-preview-8140--prefect-orion.netlify.app/ui/notifications/)

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

**Comment by WillRaphaelson** [comment]
This is great - can we add a small note on the automation event stream? When you click into an automation there is a tab for the events associated with that automation, like when its trigger was tripped or action executed?

File: docs/ui/automations.md [hunk 1]
```diff
@@ -11,48 +11,66 @@ tags:
...
 - [Details](#details) about the automation, such as a name and description.
 
-## Triggers
+### Triggers
 
 Triggers specify the conditions under which your action should be performed. Triggers can be of several types, including triggers based on: 
 
 - Flow run state change
-- Work queue health (coming soon!)
+- Work queue health
```
Comment (WillRaphaelson) [inline docs/ui/automations.md:54]: Lets add "Custom" here and link to the API section below?

**Review by WillRaphaelson — APPROVED** [review]

File: docs/ui/automations.md [hunk 2]
```diff
@@ -70,14 +88,83 @@ Similarly, if a trigger fires on work queue health and the action is to pause an
...
+```
+
+An automation reporting on work queue health might include notifications using work queue details.
+
+```
+Work queue health alert!
+
+Name: {{ work_queue.name }}
+Last polled: {{ work_queue.last_polled }}
+```
```
Comment (chrisguidry) [inline docs/ui/automations.md:166]: These are solid examples, and I think they'll be a familiar soft landing for folks familiar with Prefect but not with the Events system (pretty much everyone).  What do you think about including something like this (paraphrasing):

In addition to those shortcuts for flows, deployments, and work queues, you have access to the automation and the event that triggered your automation:

```
Automation: {{ automation.name }}
Description: {{ automation.description }}

Event: {{ event.id }}
Resource:
  {% for label, value in event.resource %}
  {{ label }}: {{ value }}
  {% %}
Related Resources:
  {% for related in event.related %}
    Role: {{ related.role }}
    {% for label, value in event.resource %}
    {{ label }}: {{ value }}
    {% %}
  {% endfor %}
```

Then something something Events API: https://app.stg.prefect.dev/api/docs#tag/Events/operation/create_workspace_events_api_accounts__account_id__workspaces__workspace_id__events_post

Comment (chrisguidry) [reply docs/ui/automations.md:166]: That also teases some more of the Jinja2 capabilities, which could help folks who are familiar with it

Comment (chrisguidry) [reply docs/ui/automations.md:166]: Below the fold is probably the right place for it :D  It's kind of in the weeds

**Review by chrisguidry — APPROVED** [review]
:heart: nice!

**Pull request merged**